docs/_tutorials/scenarios/scenarios.rst

```diff
@@ -283,6 +283,26 @@ NB: the transformation used are `pytorch.transforms classes <https://pytorch.org
 
 
 
+
+You can use TransformationIncremental along with the BackgroundSwap transform to create a domain incremental setting with changing backgrounds
+This is inspired by the Mnist meta-sets from the following `paper <https://arxiv.org/abs/2007.02915>`__.
+
+.. code-block:: python
+
```

Comment:
can you just add the imports here to have the all the classes needed please

Reply:
added